client/test/data/yofn/admin-msp/keystore/5abbeb24b1ac370db9031971a0d34abb4307dc6fc4548a7da9df4ffc4aae1c87_sk

```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
why do you need all these files in unit tests?

Reply:
@heatherlp the new code reads files and directories from a network generated by the generator, so I've checked in a generated network to test that code.

Reply:
rather than stub that out and do this in integration tests?

Reply:
@heatherlp yeah, that's a lot of file system related stubs - and we already do it for other unit tests - there's a lot of stuff under the `client/test/data` directory. All of the operations against the generated network are read only (no writes/deletes/executes).